doc/source/examples/permutation_importance_classification_leave.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>of the data instance -&gt; of the data instances</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>